src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,46 @@
+import classNames from 'classnames';
+import { FC } from 'react';
+import { FilterType } from '../../types/FilterType';
+import { Todo } from '../TodoItem/Todo';
+
+interface Props {
+  todos: Todo[],
+  filterType: FilterType,
+  onChangeFilterType: (filterType: FilterType) => void,
+}
+
+export const Footer: FC<Props> = (props) => {
+  const { todos, filterType, onChangeFilterType } = props;
+
+  const activeTodos = todos.filter(todo => !todo.completed);
+  const completedTodos = todos.filter(todo => todo.completed);
+
+  return (
+    <footer className="todoapp__footer">
+      <span className="todo-count">
+        {`${activeTodos.length} items left`}
+      </span>
+
+      <nav className="filter">
+        {Object.values(FilterType).map(value => (
+          <a
+            href={`#/${value}`}
+            key={value}
+            className={classNames('filter__link', {
+              selected: value === filterType,
+            })}
+            onClick={() => onChangeFilterType(value)}
+          >
+            {`${value.slice(0, 1).toUpperCase()}${value.slice(1)}`}
+          </a>
+        ))}
+      </nav>
+
+      {completedTodos.length > 0 && (
```

Comment:
button is a part of a flex content so if you just remova it the rest of this content will slide. In example I saw that this button fades away(like error message should).

Reply:
Grear idea! I agree, going to fix it in my last task, thanks!

Reply:
Agree. Fixed it in my final solution)